save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/controller/SandboxController.kt

```diff
@@ -60,15 +62,12 @@ class SandboxController(
         summary = "Get a list of files for provided user",
         description = "Get a list of files for provided user",
     )
-    @Parameters(
-        Parameter(name = "userName", `in` = ParameterIn.QUERY, description = "user name", required = true),
-    )
     @ApiResponse(responseCode = "200", description = "A list of files")
     @ApiResponse(responseCode = "404", description = "User with such name was not found")
     @GetMapping("/list-file")
     fun listFiles(
-        @RequestParam userName: String,
-    ): Flux<SandboxFileInfo> = blockingToMono { sandboxUserRepository.getIdByName(userName) }
+        authentication: Authentication,
+    ): Flux<SandboxFileInfo> = blockingToMono { sandboxUserRepository.getIdByName(authentication.userName()) }
```

Comment:
since `userId` is already available in `Authentication` -- can we use it instead of fetching `sandboxUserRepository` again?